Data/ByteString/Builder/ASCII.hs

```diff
@@ -373,12 +352,3 @@ intDecPadded :: P.BoundedPrim Int
 intDecPadded = P.liftFixedToBounded $ P.caseWordSize_32_64
     (P.fixedPrim  9 $ c_int_dec_padded9            . fromIntegral)
     (P.fixedPrim 18 $ c_long_long_int_dec_padded18 . fromIntegral)
-
-#else
--- compilers other than GHC
-
--- | Decimal encoding of an 'Integer' using the ASCII digits. Implemented
--- using via the 'Show' instance of 'Integer's.
-integerDec :: Integer -> Builder
-integerDec = string7 . show
```

Comment:
Could you check whether this implementation might actually be faster? The `Show Integer` instance looks fairly smart.

Reply:
foldMap integerDec (small) (10000): OK (8.14s)
        495 μs ±  19 μs, 21% slower than baseline
      foldMap integerDec (large) (10000): OK (6.84s)
         13 ms ± 381 μs, 140% slower than baseline

Reply:
Thanks! :)